src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -101,6 +110,11 @@ public ArrayList(ICollection c)
         // the internal array used to hold items.  When set, the internal 
         // array of the list is reallocated to the given capacity.
         // 
+        /// <summary>Gets or sets the number of elements that the <see cref="T:System.Collections.ArrayList" /> can contain.</summary>
+        /// <returns>The number of elements that the <see cref="T:System.Collections.ArrayList" /> can contain.</returns>
+        /// <exception cref="T:System.ArgumentOutOfRangeException"><see cref="P:System.Collections.ArrayList.Capacity" /> is set to a value that is less than <see cref="P:System.Collections.ArrayList.Count" />.</exception>
+        /// <exception cref="T:System.OutOfMemoryException">There is not enough memory available on the system.</exception>
+        /// <filterpriority>1</filterpriority>
```

Comment:
No, definitely not necessary.  We'll have to remove these before anything gets checked in (either in the intellisense files themselves or on source files the tool is run on).